.github/workflows/docker-build-deploy.yml

```
@@ -0,0 +1,140 @@
+    - name: Checkout code
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+        
+    - name: Extract version from package.json
+      id: version
+      run: |
+        VERSION=$(node -p "require('./package.json').version")
+        echo "version=$VERSION" >> $GITHUB_OUTPUT
+        echo "📦 Current version: $VERSION"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)